zerver/lib/actions.py

```diff
@@ -3661,9 +3669,28 @@ def get_emails_from_user_ids(user_ids):
     # We may eventually use memcached to speed this up, but the DB is fast.
     return UserProfile.emails_from_ids(user_ids)
 
-def realm_aliases(realm):
-    # type: (Realm) -> List[Text]
-    return [alias.domain for alias in realm.realmalias_set.all()]
+def do_get_realm_aliases(realm):
```

Comment:
I think the `do_` is for functions that change database state; `get_realm_aliases` is fine here.